changelog/undistributed/changelog_ipv6_and_asa_parsers_20210418103538.rst

```diff
@@ -0,0 +1,35 @@
+--------------------------------------------------------------------------------
+                                NEW
+--------------------------------------------------------------------------------
+* IOSXE
+  * New command parsers:
+    * show ipv6 eigrp interfaces
+    * show ipv6 eigrp interfaces detail
+    * show ip eigrp interfaces detail
+    * show ipv6 protocols
+    * show key chain
+  * Modified command parsers:
+    * show ip eigrp interfaces
+      * Adjusting p1 regex to support IPv6 too
+      * Offloading parser to a SuperParser class
+      * support eigrp named mode
+      * adding optional keys to ShowIpEigrpInterfacesSchema schema to support `show ip eigrp interfaces detail parser`
+* IOS
+  * New command parsers:
+    * show ipv6 eigrp interfaces
+    * show ipv6 eigrp interfaces detail
+    * show ip eigrp interfaces detail
+    * show ipv6 protocols
+    * show key chain
+  * Modified command parsers:
+    * show ip eigrp interfaces
+      * Adjusting p1 regex to support IPv6 too
+      * Offloading parser to a SuperParser class
+      * Support eigrp named mode
+      * adding optional keys to ShowIpEigrpInterfacesSchema schema to support `show ip eigrp interfaces detail parser`
+* ASA
+  * New command parsers:
+    * show crypto ikev2 sa
+    * show nameif
+    * show failover
+    * show failover interface
```

Comment:
It's possible that our format for changelogs for the genieparser repo may change very soon. If so, then I'll re-write the changelog for you.